### PR TITLE
Fix styles on dashboard company list button

### PR DESCRIPTION
**Pull request by ian-leggett**
## Description of change
The buttons in the company lists inside the dashboard are inheriting an active state CSS rule from a deprecated CSS file called `_dashboard.scss` which lives in `/assets/stylesheets/_deprecated/_dashboard.scss`. 

See below:
Screenshot 2020-03-31 at 16 30 29

We didn't notice this before as we hadn't seen any active states until after it was released. The solution was to remove the class that was on a containing div called `dashboard-section` so we don't inherit these style rules from the old deprecated file. After doing this I needed to style the font-sizes in the referral list cards as they were also inheriting rules from the old file.

 
## Test instructions

No active style states should show in the company lists buttons/links. Also referrals should look like it did before.

## Screenshots
Screenshot 2020-03-31 at 16 30 29

### After

Screenshot 2020-03-31 at 16 38 29
Screenshot 2020-03-31 at 16 39 34

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to master?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [x] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "integrationFolder": "test/functional/cypress/specs",
-  "baseUrl": "http://localhost:3001",
+  "baseUrl": "http://localhost:3000",
```
Comment (web-bert) [inline cypress.json:3]: is this intentional?

File: src/client/components/Dashboard.jsx [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
+import styled from 'styled-components'
+import { GREY_2 } from 'govuk-colours'
 
 import CompanyLists from './CompanyLists'
 import ReferralList from './ReferralList'
 import TabNav from './TabNav'
 
+const StyledDiv = styled('div')`
+  border-top: 4px solid ${GREY_2};
+  padding-top: 16px;
```
Comment (web-bert) [inline src/client/components/Dashboard.jsx:12]: Should this be using `SPACING`?

File: cypress.json (hunk 1 above)
Comment (ian-leggett) [reply cypress.json]: Good spot, sorry this was a mistake. I have put this back 👍

File: src/client/components/Dashboard.jsx (hunk 2 above)
Comment (ian-leggett) [reply src/client/components/Dashboard.jsx:12]: I did start to use SPACING but then realised that the spacing from the container div and class I removed ie `dashboard-section` had a padding of 16px and this does not fall into the SPACING units available in constants. Maybe the dashboard section which uses legacy styles needs updating so both the dashboard section and the new React dashboard can use SPACING.

Comment (web-bert) [reply src/client/components/Dashboard.jsx:12]: Is a 1px difference going to be noticable?

**Pull request merged**